### DISCUSSION TITLE
Exercise 4.4.20

### DISCUSSION THREAD

**Discussion by HamzaIbrahimi** [spamegg1/Epp-Discrete-Math-5th-solutions#17] [General]
Have an exam so I'm just going through some exercises. But for the proof the simplification of the expression should be:

$n + n + 1 + n + 2 = 3n + 3$

It seems the answer provided, $3n + 6$, is incorrect.

**Comment by spamegg1** [comment]
Thanks for reporting! I'll fix it.

**Comment by spamegg1** [comment]
Fixed in #21
